src/components/Notification.tsx

```diff
@@ -0,0 +1,217 @@
+      <div
+        className="bg-[rgba(0,0,0,0.45)] w-full h-full absolute top-0 left-0 z-1"
+        role="button"
+        aria-label="background"
+        tabIndex={0}
+        onClick={handleShowNotification}
+        onKeyDown={handleShowNotification}
+        data-testid="keyClick"
+      />
+      <div className="absolute top-[60px] right-0 left-0 mx-auto px-2 md:mx-0 md:left-auto md:right-[80px] z-2  w-full max-w-[392px] h-[calc(100%-70px)]">
+        <div className="flex flex-col flex-nowrap w-full h-max max-h-full bg-[#E5EAFF] dark:bg-dark-tertiary rounded-[20px]">
+          <div className="w-full p-3 border-[rgba(95,91,91,0.5)] dark:border-white border-b-[0.5px]">
+            <p className="font-semibold dark:text-white">{t('Notification')}</p>
+          </div>
+          <div
+            className="flex flex-col w-full overflow-auto"
+            data-testid="notificationsContainer"
+          >
+            {notifications.map((notification, index) => (
+              <div
+                className="w-full p-5 border-[rgba(95,91,91,0.5)] dark:border-white border-b-[0.5px]"
+                key={notification.id}
+              >
+                <div className="flex flex-row justify-between align-center gap-x-[20px] ">
+                  <img
+                    src={notification.profile}
+                    alt="oldMan"
+                    className="rounded-[1000px] w-[60px] h-[60px] object-cover cursor-pointer"
+                  />
+
+                  <div
+                    className="flex flex-col w-full gap-[5px] cursor-pointer"
+                    onClick={() => {
+                      markRead(
+                        notification.id,
+                        notifications,
+                        setNotifications,
+                      );
+                    }}
+                    data-testid={index === 0 && 'read'}
+                  >
+                    <p className="font-bold dark:text-white">
+                      {notification.names}
+                    </p>
+                    <p className="text-[#111827] dark:text-white text-[12px]">
+                      {notification.description}
+                    </p>
+                    <p className="text-[12px] dark:text-white">
+                      {format(new Date(notification.created), 'MMMM dd, p')}
+                    </p>
+                  </div>
+
+                  <div className="flex flex-col items-center transition-all">
+                    <div
+                      className={`h-[10px] w-[10px] rounded-full ${
+                        !notification.read
+                          ? 'bg-[#148FB6]'
+                          : 'border-[rgba(95,91,91,0.5)] dark:border-white border-[1px]'
+                      }  mt-[7px] mb-[10px]`}
+                    />
+                    <XIcon
+                      className="border-[rgba(95,91,91,0.5)] dark:fill-white h-[20px] w-[20px] cursor-pointer"
```

Comment:
Avoid absolute values in the styling, instead, put the color in our `tailwind.css.config` to make it easier to replace the colors when the time comes